### PR TITLE
Always use svg display in gis editor

### PR DESCRIPTION
**Pull request by MoonE**
Editing geometries with SRID should display with OpenLayers visualization. As that doesn't work this now always uses the svg viewer.

Fixing the OpenLayers visualization here would be rather complicated.

Will provide a complete fix for master branch shortly ...

### CODE REVIEW

**Review by williamdes — APPROVED** [review]
This is an acceptable workaround but feels weird
Is there no way it could be fixed in a proper way, is it too broken?

**Comment by MoonE** [comment]
I'll have a look.

File: templates/gis_data_editor_form.twig [hunk 1]
```diff
@@ -11,14 +11,16 @@
         {{ get_hidden_inputs() }}
 
         {# Visualization section #}
-        <div id="placeholder"{{ srid != 0 ? ' class="hide"' }}>
+        <!-- Always uses svg-view here added `and false` as a quick-fix -->
+        <div id="placeholder"{{ srid != 0 and false ? ' class="hide"' }}>
```
Comment (MauricioFauth) [inline templates/gis_data_editor_form.twig:15]:
```suggestion
        <div id="placeholder">
```

File: templates/gis_data_editor_form.twig [hunk 2]
```diff
@@ -11,14 +11,16 @@
...
             {{ visualization|raw }}
         </div>
 
         <div id="openlayersmap" style="width: {{width}}px; height: {{height}}px;"{{ srid == 0 ? ' class="hide"' }}></div>
 
         <div class="choice float-end">
-            <input type="checkbox" id="choice" value="useBaseLayer"{{ srid != 0 ? ' checked="checked"' }}>
+            <!-- Always uses svg-view here added `and false` as a quick-fix -->
+            <input type="checkbox" id="choice" value="useBaseLayer"{{ srid != 0 and false ? ' checked="checked"' }}>
```
Comment (MauricioFauth) [inline templates/gis_data_editor_form.twig:23]:
```suggestion
            <input type="checkbox" id="choice" value="useBaseLayer">
```

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

**Review by MauricioFauth — APPROVED** [review]

**Pull request merged**